lib/include/tick/hawkes/inference/hawkes_cumulant.h

```diff
@@ -35,4 +33,29 @@ class DLL_PUBLIC HawkesCumulant : public ModelHawkesList {
   }
 };
 
+class DLL_PUBLIC HawkesTheoreticalCumulant {
+ private:
+  int d;
+  SArrayDoublePtr mu;
+  SArrayDoublePtr Lambda;
+  SArrayDouble2dPtr C;
+  SArrayDouble2dPtr Kc;
+  SArrayDouble2dPtr R;
+
+ public:
+  HawkesTheoreticalCumulant(int);
+  int get_dimension() { return d; }
+  void set_baseline(const SArrayDoublePtr mu) { this->mu = mu; }
```

Comment:
We might not have it everywhere, but `T const& n` is the preferred order

Reply:
Changed: fa43f6dc9dc65013c609508c0f4fe43290fe60ac